feature-configs/e2e-gcp/sctp/sctp_module_mc_patch.yaml

```diff
@@ -2,5 +2,5 @@ apiVersion: machineconfiguration.openshift.io/v1
 kind: MachineConfig
 metadata:
   labels:
-    machineconfiguration.openshift.io/role: worker
+    machineconfiguration.openshift.io/role: worker-sctp
```

Comment:
I think this needs to be updated... worker-sctp changed to worker-cnf

Reply:
Done